app/components/status-banner/component.ts

```diff
@@ -0,0 +1,13 @@
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+
+import { StatusMessage } from 'ember-osf-web/services/status-messages';
+
+export default class StatusBanner extends Component.extend({
+    didReceiveAttrs() {
+        this.set('messages', this.get('statusMessages').getMessages());
```

Comment:
Oh, didn't think about it earlier, but this should go in `constructor()` (and can get rid of `.extend()`!). Since there are no attrs to receive, it'll be equivalent.

Reply:
Isn't it true that didReceiveAttrs gets called on every re-render, even when no attributes are passed? As in, a link-to to a page and then back would trigger this again (which is the wanted behavior), while the constructor would only trigger once?